From 759c0bc6f990240287a14fa6dec029ecea581463 Mon Sep 17 00:00:00 2001
From: Arun Raghavan <arun.raghavan@collabora.co.uk>
Date: Sat, 27 Apr 2013 03:27:05 +0000
Subject: build-sys: json-c changed everything :(

json-c 0.11 ships a pkg-config file with a different name, and changes
the name of its library as well.
---
(limited to 'configure.ac')

diff --git a/configure.ac b/configure.ac
index c8161a4..b1bd21a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -624,7 +624,8 @@ AM_CONDITIONAL([HAVE_TESTS], [test "x$HAVE_LIBCHECK" = x1])
 
 #### json parsing ####
 
-PKG_CHECK_MODULES(LIBJSON, [ json >= 0.9 ])
+PKG_CHECK_MODULES(LIBJSON, [ json-c >= 0.11 ], [],
+		  [PKG_CHECK_MODULES(LIBJSON, [ json >= 0.9 ])])
 AC_SUBST(LIBJSON_CFLAGS)
 AC_SUBST(LIBJSON_LIBS)
 
--
cgit v0.9.0.2-2-gbebe
